src/subcommand.rs

```diff
@@ -0,0 +1,81 @@
+use crate::common::*;
+
+#[derive(Debug, PartialEq, StructOpt)]
+pub(crate) enum Subcommand {
+    Download {
+        #[structopt(name = "OUT", long = "out", short = "o", default_value = "dump")]
+        out: PathBuf,
+
+        #[structopt(name = "NUMBER", long = "number", short = "n", use_delimiter(true))]
+        number: Vec<u32>,
+    },
+    Bottleneck {
```

Comment:
I just don't like subcommand being a noun. It makes more sense when you call ``./asmap-rs find_bottleneck`` rather than ``./asmap-rs bottleneck``, isn't it? See verbs for subcommands in this [example](https://docs.rs/clap/2.24.1/clap/struct.ArgMatches.html#method.subcommand).

Reply:
I prefer `Bottleneck`, but I'll survive with `FindBottleneck`. Will change it :)